### PR TITLE
Frontend for moving away from collection ids, and just using the name…

### PR DESCRIPTION
**Pull request by andrewconner**
Fixes a bug where if you search for a tag, you can't rename or delete it. @cvializ

### CODE REVIEW

**Comment by andrewconner** [comment]
Uses new backend from #18842, tested as working.

File: kifi-backend/modules/shoebox/angular/src/tagManage/tagManage.js [hunk 1]
```diff
@@ -196,8 +196,8 @@ angular.module('kifi')
         template: 'tagManage/modifyTagProgressModal.tpl.html',
         scope: _scope
       });
+      manageTagService.rename($scope.selectedTag.name, $scope.selectedTag.renamed);
       $scope.selectedTag.name = $scope.selectedTag.renamed;
```
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/tagManage/tagManage.js:200]: what if the rename is unsuccessful?

Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/tagManage/tagManage.js:200]: I suppose I can rename it back if the server gives an explicit error. It issue is that it takes a while... has to edit and reprocess the notes of every keep that has the tag. Seems to do about 15-30 a second. So if you have more than a few hundred, the request will time out (but it's still running on the backend).

Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/tagManage/tagManage.js:200]: mmm I see. lgtm then

**Comment by cvializ** [comment]
:+1:

**Pull request merged**